sdks/python/apache_beam/ml/inference/sklearn_inference.py

```diff
@@ -26,8 +26,7 @@
 
```

Comment:
Might have to merge this file since I just merged a change. Should be straight forward.